app/src/main/java/nz/org/cacophony/birdmonitor/Util.java

```diff
@@ -234,14 +234,18 @@ public static File getRecordingsOrNotesFolder(Context context, String recordings
      * @return
      * @throws Exception
      */
-    public static String getDeviceID(String webToken) throws Exception {
+    public static long getDeviceID(String webToken) throws Exception {
         if (webToken == null) {
-            return "";
+            return 0;
         }
 
         String webTokenBody = Util.decoded(webToken);
         JSONObject jObject = new JSONObject(webTokenBody);
-        return jObject.getString("id");
+        try {
+            return Long.parseLong(jObject.getString("id"));
+        }catch (NumberFormatException ex){
```

Comment:
space after `}` and `)`

Reply:
Sorry need to pay more attention to these